packages/volto-slate/news/7182.bugfix

```diff
@@ -0,0 +1,5 @@
+Improved volto-slate package:
+- Solved: the condition for making an initial empty paragraph inside a RichTextWidget was wrong.
```

Comment:
Why was it wrong, or alternatively, what is the right way, or better yet both? Here's a template as an example.

The condition for making an initial empty paragraph inside a RichTextWidget now does X, whereas it previously incorrectly did Y.